### PR TITLE
Feature#28 store and employees database edit delete

### PR DESCRIPTION
**Pull request by EverardoAlcarazBaltazar**
# Description

What did you implemented/Why did you implemented this:

Edit and delete function from the employee and store tables
The database was linked to the form of employee and store.

## Screenshots
edit
employe_regiser_funtion
tabla de stores

### CODE REVIEW

File: app/assets/stylesheets/pages/_crud_employee.css.scss [hunk 1]
```diff
@@ -36,6 +36,12 @@
```
Comment (dagp-ucol) [inline app/assets/stylesheets/pages/_crud_employee.css.scss:36]: #### Incorrect Identation

#### Some blocks are not ordered alphabetically.

image

## 
image

File: app/assets/stylesheets/pages/_crud_store.css.scss [hunk 2]
```diff
@@ -122,6 +122,11 @@
             top: 295px;
             transition: all 0.3s ease 0s;
             width: 60px;
+
+            a {
+              text-decoration: none;
+              color: $font-color-gray;
```
Comment (dagp-ucol) [inline app/assets/stylesheets/pages/_crud_store.css.scss:128]: Order alphabetically.

File: app/assets/stylesheets/pages/_crud_store.css.scss [hunk 3]
```diff
@@ -139,6 +144,11 @@
             top: 295px;
             transition: all 0.3s ease 0s;
             width: 59px;
+
+            a {
+              text-decoration: none;
+              color: $font-color-gray;
```
Comment (dagp-ucol) [inline app/assets/stylesheets/pages/_crud_store.css.scss:150]: Order alphabetically.

**Review by dagp-ucol — CHANGES_REQUESTED** [review]

File: app/controllers/employees_controller.rb [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+    if @employee.update(employee_params)
+      redirect_to '/employees'
+    else
+      render :edit, status: :unprocessable_entity
+    end
+  end
+
+  def destroy
+    @employee = Employee.find(params[:id])
```
Comment (juperameza) [inline app/controllers/employees_controller.rb:42]: You can refactor by doing something like this 
`def set_employee
      @employee = Employee.find(params[:id])
   end`

What do you think

File: app/controllers/stores_controller.rb [hunk 5]
```diff
@@ -20,6 +20,27 @@ def create
...
+  
+      if @store.update(store_params)
+        redirect_to '/stores'
+      else
+        render :edit, status: :unprocessable_entity
+      end
+    end
+  
+    def destroy
+      @store = Store.find(params[:id])
```
Comment (juperameza) [inline app/controllers/stores_controller.rb:39]: Same.here with the refactor

File: app/views/employees/new.html.erb [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+          <%= form.label :private_number %><br>
+          <%= form.text_field :private_number %>
+        </div>
+        <div class="button-container">
+          <%= form.submit %>
+        </div>
+      <% end %>
+    </section>
+  </div>
+</div>
```
Comment (juperameza) [inline app/views/employees/new.html.erb:34]: Add a blank line

File: app/views/employees/edit.html.erb [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+          <%= form.label :private_number %><br>
+          <%= form.text_field :private_number %>
+        </div>
+        <div class="button-container">
+          <%= form.submit %>
+        </div>
+      <% end %>
+    </section>
+  </div>
+</div>
```
Comment (juperameza) [inline app/views/employees/edit.html.erb:34]: Add a blank line at the end 
Also aren't edit and new forms the same what do you think about using [partial](https://guides.rubyonrails.org/getting_started.html#rendering-a-partial-form)

File: app/views/stores/edit.html.erb [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+        <div class="info-container">
+          <%= form.label :adress %><br>
+          <%= form.text_field :adress %>
+        </div>
+        <div class="button-container">
+          <%= form.submit %>
+        </div>
+      <% end %>
+    </section>
+  </div>
```
Comment (juperameza) [inline app/views/stores/edit.html.erb:21]: Blank line at the end of this file

**Review by juperameza — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/pages/_crud_employee.css.scss (hunk 1 above)
Comment (DiegoMoralesBravo) [reply app/assets/stylesheets/pages/_crud_employee.css.scss:37]: Done

File: app/assets/stylesheets/pages/_crud_store.css.scss (hunk 3 above)
Comment (DiegoMoralesBravo) [reply app/assets/stylesheets/pages/_crud_store.css.scss]: Done

File: app/assets/stylesheets/pages/_crud_store.css.scss (hunk 2 above)
Comment (DiegoMoralesBravo) [reply app/assets/stylesheets/pages/_crud_store.css.scss:140]: Done

**Review by dagp-ucol — APPROVED** [review]

**Review by oscaregr — APPROVED** [review]
IANS

**Review by juperameza — APPROVED** [review]
Lgtm

**Pull request merged**